internal/k8s/runtime_test.go

```diff
@@ -0,0 +1,154 @@
+package k8s
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path"
+	"testing"
+
+	"github.com/Shopify/kubeaudit/k8stypes"
+	"github.com/stretchr/testify/assert"
+)
+
+const fixtureDir = "../test/fixtures"
+
+func TestNewTrue(t *testing.T) {
+	assert.True(t, *NewTrue())
+}
+
+func TestNewFalse(t *testing.T) {
+	assert.False(t, *NewFalse())
+}
+
+func TestEncodeDecode(t *testing.T) {
+	assert := assert.New(t)
+
+	deployment := k8stypes.NewDeployment()
+	deployment.ObjectMeta = k8stypes.ObjectMetaV1{Namespace: "foo"}
+	deployment.Spec.Template.Spec.Containers = []k8stypes.ContainerV1{{Name: "bar"}}
+
+	expectedManifest := `apiVersion: apps/v1
+kind: Deployment
```

Comment:
Nitpick: can this block be indented twice so it's properly set with everything else?

Reply:
Hmm It's a string comparison so we would need to add indents to the programatically produced manifest for them to still be equal, or like take out the indents before comparing. I could move the variable out of the function so the variable name is at least indented the same level as the rest of the file? What do you think?

Reply:
Ah right, makes sense. Can you move the `apiVersion: apps/v1` onto a newline so it's indented the same level as the rest of the expected manifest?

Reply:
We would have to prepend a newline to the programatically produced output but yes we could do that. Do you think it makes sense to have the "expected output" in the test be different from the "actual expected output" though? As in, we don't want the manifest to start with a newline (because that would be an empty line at the top of the file), but we would be setting that as the expected output in the test for formatting purposes.

Reply:
What do you think about moving it into a yaml file?

Reply:
I'm not too familiar with idiomatic Golang when it comes to having fixtures, but I think having a fixtures directory to keep this new yaml file (along with any other ones we have now and/or in the future) will be beneficial?

Reply:
I'm honestly not too sure either. The old version (before the refactor) had a single fixtures directory and I personally thought that was kind of a mess, so I created a fixtures directory for each auditor. I think having a fixtures directory inside any directory that has test files is fine though. What do you think?

Reply:
Yeah, this makes sense to me!